istio-telemetry/mixer-telemetry/values.yaml

```diff
@@ -53,6 +53,15 @@ mixer:
     stackdriver:
       enabled: false
 
+      auth:
+        appCredentials: false
+        apiKey: ""
+        serviceAccountPath: ""
+
+      tracer:
+        enabled: false
+        sampleProbability: 1
```

Comment:
We already have a value `traceSampling` I think

Reply:
I know this is confusing, but the confusion is a bit stemming from the API itself. Pilot has a traceSampling field to set this and its value ranges from 0 to 100. 

For this sampleProbability, the range is between 0-1, and acts like a second level filter after what pilot enforces proxies. That said, I'm ok with removing this field completely and set the value to 1 by default.